README.md

```diff
@@ -43,12 +45,42 @@ It deploys a hello-openshift application 3 times:
 
 
 
-## Bootstrapping
+## Setup instructions
+
+### Default single cluster setup with `values-simple.yaml`
+
+#### Configuring required secrets / parameters
+The secrets here secure Trustee and the peer-pod vms. Mostly they are for demonstration purposes. 
+This only has to be done once.
 
-#### Install of OCP cluster on azure.
+1. `cp ./values-secret.yaml.template ~/values-secret-coco-pattern.yaml`
+2. Examine the sample secrets e.g. `kbsRes1` and `passphrase` and set as appropriate or random secrets will be generated in vault
+3. If you have not previously, run the following:
+   1. `sh ./scripts/create-default-policy.sh`
+   2. `sh ./scripts/gen-kbs-keys.sh`
+   3. `sh ./scripts/gen-ssh-key-azure.sh`
 
+#### Install on an OCP cluster on azure using Red Hat Demo Platform
+
+Red Hat a demo platform. This allows easy access for Red Hat associates and partners to ephemeral cloud resources. The pattern is known to work with this setup.
+1. Get the [openshift installer](https://console.redhat.com/openshift/downloads)
+   1. **NOTE: openshift installer must be updated regularly if you want to automatically provision the latest versions of OCP**
+2. Get access to an [Azure Subscription Based Blank Open Environment](https://catalog.demo.redhat.com/catalog?category=Open_Environments&search=azure&item=babylon-catalog-prod%2Fazure-gpte.open-environment-azure-subscription.prod).
+3. Import the required azure environmental variables (see coded block):
+   ```
+      export CLIENT_ID=
+      export PASSWORD=
+      export TENANT=
+      export SUBSCRIPTION=
+      export RESOURCEGROUP=
+  ```
+1. Run the wrapper install script 
+  1. `python ./rhdp/rhdp-cluster-define.py`
```

Comment:
You should probably bail out in the script if any of these env variables are not set to avoid potential mistakes happening